lib/rspec/core/example_group.rb

```diff
@@ -33,6 +33,8 @@ class ExampleGroup
       include Pending
       extend SharedExampleGroup
 
+      # Define a singleton method for the eigenclass (remove the method if
+      # it's already been defined).
```

Comment:
No. but if anyone like me ever wanted to do a code review he/she probably gets stuck in the process.